src/screens/SplashScreen/index.js

```diff
@@ -48,9 +49,16 @@ const SplashScreen = () => {
     try {
       const id = await getUserId();
       const anonymousUserId = await getAnonymousUserId();
-      setProfileId({
+      const token = TokenStorage.get(ITokenEnum.token);
+      const anonymousToken = TokenStorage.get(ITokenEnum.anonymousToken);
+
+      console.log('doVerifyUser', id, anonymousUserId, token, anonymousToken);
+
+      setAuth({
         anonProfileId: anonymousUserId,
-        profileId: id
+        profileId: id,
+        token,
+        anonymousToken
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the code seems fine, it's important to consider error handling for `TokenStorage.get()`. If there's a possibility that this function could throw an error (for example, if the token doesn't exist in storage), you should wrap these calls in a try-catch block to prevent the application from crashing.
```diff
+    try {
+      const id = await getUserId();
+      const anonymousUserId = await getAnonymousUserId();
+      const token = TokenStorage.get(ITokenEnum.token);
+      const anonymousToken = TokenStorage.get(ITokenEnum.anonymousToken);
+
+      console.log('doVerifyUser', id, anonymousUserId, token, anonymousToken);
+
+      setAuth({
+        anonProfileId: anonymousUserId,
+        profileId: id,
+        token,
+        anonymousToken
+      });
+    } catch (error) {
+      console.error("Error retrieving tokens:", error);
+    }
```
Also, please ensure that logging sensitive information like tokens is acceptable in your security policy. If not, remove the console log statement that logs these values.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
done on latest commit